app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -2115,6 +2166,7 @@ private void showExternalPlaybackDialog() {
      * Remove unneeded information while waiting for a next task
      * */
     private void cleanUp() {
+        toggleFullscreenIfInFullscreenMode();
```

Comment:
This was added so that when the `VideoDetailFragment` is disposed/reused it always starts with the device orientation and not in fullscreen? Please add a comment

Reply:
Added a comment